quipucords/api/credential/view_v1.py

```diff
@@ -83,3 +85,48 @@ def destroy(self, request, pk):
 
         except Credential.DoesNotExist as exception:
             raise Http404 from exception
+
+
+@api_view(["post"])
+def credential_bulk_delete(request) -> Response:
+    """
```

Comment:
I like that the destructive part was moved to model, but why was this view moved from `view.py` to `view_v1.py`? 🤔

Reply:
This `api_view` is what powers the original `/v1/credentials/bulk_delete/`, which you can see in `urls.py`:

```
v1_urls = [
    path(
        "credentials/bulk_delete/",
        credential_bulk_delete,
        name="credentials-bulk-delete",
    ),
```

For v2, there's a new method `CredentialViewSetV2.bulk_delete` that effectively does the same thing.

Reply:
…which might not be obvious at first because making `CredentialViewSetV2.bulk_delete` _part of the viewset class_ means the v2 bulk delete does not need an explicit custom addition to the urls list like the old `api_view` function did. The v2 bulk delete just automatically works when requests invoke the path on the viewset.

Reply:
aaaah, I missed it. TY.